### ISSUE TITLE
Secure map editor against invalid access

### ISSUE THREAD

**Issue by ThorbenStaufer** [ElektraInitiative/PermaplanT#480]
Currently every user could open any map through entering random map IDs in the URL.
The map editor should check on init, if the currently logged-in user is the map creator or if the map allows other contributors.

**Comment by markus2330** [comment]
Absolutely, very good observation. I think we should additionally use UUID (nearly) everywhere to even avoid that ids can be guessed or that there are problems in concurrent creation. I added some text about it in e5de3a7ba90989a21776402a8ade20e5588a49db

Let us discuss this today.
